config/pepperConfig.js.ctmpl

```diff
@@ -259,3 +259,40 @@ var DDP_ENV = {
         {{end}}
     {{end}}
 {{end}}
+
+{{if eq $study_guid "circadia"}}
+    {{if eq $environment "local"}}
+        DDP_ENV['auth0ClientId'] = "GMMFtLjB1c38Fmr3UVEm8WqBsvCIeJ9d";
+        DDP_ENV['auth0Domain'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['auth0Audience'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+    {{end}}
+
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "tbDahmqfZo7ircGJ6AGsBrtXXGgA29Pk";
+        DDP_ENV['auth0Domain'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['auth0Audience'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+    {{end}}
+
+    {{if eq $environment "test"}}
+        DDP_ENV['auth0ClientId'] = "TBD";
```

Comment:
I recommend we read some of these values out from Vault. That way, we can just set the values in Vault instead of having to make a new code commit when we push to test/staging/prod. See RGP above for an example.

Reply:
That's a good idea. I'll address that. Thanks!